gitops/azure-devops/MultipleServices.md

```diff
@@ -0,0 +1,97 @@
+# Operating Multiple Services
+
+> *Please first overview the [Pipeline Thinking](./PipelineThinking.md) document before continuing through this document.*
+
+## Overview: GitOps
+
+In our prescribed gitops workflow, you are expected to push changes to your cluster through changes to a HLD repository.
+
+Recall the prescribed workflow that encompasses the manifest yaml generation pipeline:
+
+![Example of manifest yaml generation pipeline](images/manifest-gen.png)
```

Comment:
Unable to load this image?

Reply:
might be the absolute path vs a relative path